t/t2801-top-cmd.t

```diff
@@ -50,28 +75,63 @@ test_expect_success 'flux-top fails if TERM is not supported' '
 	test_must_fail $runpty --term=dumb --stderr=dumb.err flux top &&
 	grep "terminal does not support required capabilities" dumb.err
 '
-test_expect_success 'flux-top --test-exit works with a pty' '
-	$runpty flux top --test-exit >/dev/null
-'
 test_expect_success 'submit batch script and wait for it to start' '
 	cat >batch.sh <<-EOT &&
 	#!/bin/sh
 	touch job2-has-started
-	flux mini run sleep 300
+	jobid=\$(flux mini submit --flags waitable sleep 300)
+	flux job wait-event \$jobid start
+	touch job2-batch-sleep-has-started
+	flux job wait --all
```

Comment:
Oh, also, a more idiomatic way to wait for all jobs to complete at the end of a batch script would be
```
flux queue drain
```
Then you don't need to add the `waitable` flag to all jobs.

Reply:
oh that's a good idea